cli-commands/forgae-deploy/forgae-deployer.js

```diff
@@ -234,7 +236,17 @@ async function generateFunctionsFromSmartContract(contractPath, deployedContract
 
             let resultFromExecution = await client.contractCall(byteCode, ABI_TYPE, deployedContract.address, myName, configuration);
 
-            return (await resultFromExecution.decode(myReturnType)).value;
+            let returnType = myReturnType;
```

Comment:
rename "myReturnType" to something more context oriented. For example functionReturnType or something